src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -18,7 +16,7 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Person person) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+                .anyMatch(keyword -> person.getName().fullName.toLowerCase().contains(keyword.toLowerCase()));
```

Comment:
Maybe you can change this in the `StringUtil::containsWordIgnoreCase` function to maintain the same level of abstraction?

Reply:
Fixed in the latest commit :)